ui/src/config/section/compute.js

```diff
@@ -65,9 +65,15 @@ export default {
         if (store.getters.userInfo.roletype === 'Admin') {
           fields.splice(2, 0, 'instancename')
           fields.push('account')
+          if (store.getters.listAllProjects) {
```

Comment:
@hsato03 
project is available for all types of users (root, domain admin  and users). 
the code block can be moved to line 64
```
          if (store.getters.listAllProjects) {
            fields.push('project')
          }
```

Reply:
Account are also available for all types of users, but the `account` column is only displayed for root and domain admins. I guess it would be better to keep the current behavior since the project is an account type.

Reply:
@hsato03 
I was not talking about the `account`, but `project`
`project` can be displayed  not only for root and domain admins, but also regular users

Reply:
@weizhouapache I think you are right, as regular users can create a `project` but not an `account` by default. I will work to change this behavior.